### PR TITLE
CB-6125 creator of an environment will automaticly get enviromentAdmin

### PR DESCRIPTION
**Pull request by schfeca75**
@keyki or @horadla23 pls review

### CODE REVIEW

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java [hunk 1]
```diff
@@ -59,6 +59,19 @@ public void cleanupVirtualGroups(String accountId, String environmentCrn) {
...
 
+    public void assignEnvironmentAdminRole(String userCrn, String environmentCrn) {
+        try {
+            grpcUmsClient.assignResourceRole(userCrn, environmentCrn, grpcUmsClient.getBuiltInEnvironmentAdminResourceRoleCrn(), MDCUtils.getRequestId());
+            LOGGER.debug("EnvironmentAdmin role of {} environemnt is assigned to the {} user", environmentCrn, userCrn);
+        } catch (StatusRuntimeException ex) {
+            if (Code.ALREADY_EXISTS.equals(ex.getStatus().getCode())) {
+                LOGGER.debug("EnvironmentAdmin role of {} environemnt is already assigned to the {} user", environmentCrn, userCrn);
+            } else {
+                throw ex;
```
Comment (keyki) [inline auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java]: I don't think that we should make the provisioning fail here. Simply logging the error would do.

File: environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java [hunk 2]
```diff
@@ -106,6 +106,8 @@ public EnvironmentDto create(EnvironmentCreationDto creationDto) {
             // To keep backward compatibility, if somebody passes the group name, then we shall just use it
             environmentService.setAdminGroupName(environment, creationDto.getAdminGroupName());
         }
+        virtualGroupService.assignEnvironmentAdminRole(creationDto.getCreator(), environmentCrnForVirtualGroups);
```
Comment (keyki) [inline environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java]: This is not related to virtual group at all. It should be moved to some other proper place.

**Comment by keyki** [comment]
There is no un-assignment here. I don't know if it's needed or not. @horadla23 might know.

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java [hunk 3]
```diff
@@ -59,6 +59,19 @@ public void cleanupVirtualGroups(String accountId, String environmentCrn) {
         }
     }
 
+    public void assignEnvironmentAdminRole(String userCrn, String environmentCrn) {
+        try {
+            grpcUmsClient.assignResourceRole(userCrn, environmentCrn, grpcUmsClient.getBuiltInEnvironmentAdminResourceRoleCrn(), MDCUtils.getRequestId());
+            LOGGER.debug("EnvironmentAdmin role of {} environemnt is assigned to the {} user", environmentCrn, userCrn);
+        } catch (StatusRuntimeException ex) {
+            if (Code.ALREADY_EXISTS.equals(ex.getStatus().getCode())) {
```
Comment (horadla23) [inline auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java]: I didn't know about this code, I would move this exception handling into GrpcUmsClient assignResourceRole method

File: environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java (hunk 2 above)
Comment (horadla23) [reply environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java]: I agree, assignment logic should be in the service of the resource, in this case in some EnvironmentService class I assume

Comment (schfeca75) [reply environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java]: Moved to the environmentService

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java (hunk 3 above)
Comment (schfeca75) [reply auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java]: I would  left this code here as it is specific during environment creation, maybe later for other resources it could be an error case.

Comment (horadla23) [reply auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java]: ok, it is fine for me
